_posts/2019-04-04-docker-vms-and-tech-portfolio.markdown

```diff
@@ -3,6 +3,7 @@ layout: post
 title: "Docker on VMs and the Technology Portfolio"
 date: 2019-04-04 09:30
 categories: upcoming
```

Comment:
Please set this to `categories: meetings`, so the meeting no longer shows as "upcoming" on the homepage.

Reply:
I made that change in #85